views/layouts/default.hbs

```diff
@@ -1,18 +1,17 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <link rel="stylesheet" href="https://code.cdn.mozilla.net/fonts/fira.css">
+    <link rel="stylesheet" href="/css/app.css">
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0, viewport-fit=cover" />
     <meta name="description" content="Firefox Monitor notifies users when their credentials have possibly been leaked or stolen in a data breach. Powered by haveibeenpwned.com." />
     <meta property="og:title" content="Firefox Monitor" />
-    <meta name="twitter:title" content="Firefox Monitor" />
     <meta property="og:description" content="Protect yourself from hackers." />
-    <meta name="twitter:description" content="Protect yourself from hackers." />
-    <meta property="og:url" content="https://monitor.firefox.com" />
-    <meta property="og:image" content="https://monitor.firefox.com/img/firefox-share.png" />
+    <meta property="og:url" content="https://fx-breach-alerts.herokuapp.com/" />
+    <meta property="og:image" content="/img/firefox-share.png" />
```

Comment:
I _think_ these need to be absolute URLs for when Twitter/Facebook scrapes the content and tries to embed it in a Card.

It'd be nice if we could specify some base URL via an env variable (like the `og:url`) and just use that as the prefix.

Reply:
We can do this (and the upcoming `STATIC_URL_HOST`) like so:

`server.js`:
```js
app.locals.SERVER_URL = AppConstants.SERVER_URL;
```

And then in `default.hbs`:
```html
<meta content="{{ SERVER_URL }}" />
```